<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerSense</class>
 <widget class="QMainWindow" name="LayerSense">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1354</width>
    <height>952</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="20,20,2,8">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QChartView" name="graphics_Dynamic"/>
      </item>
      <item row="0" column="1">
       <widget class="QChartView" name="graphics_Velocity"/>
      </item>
      <item row="0" column="2">
       <widget class="QChartView" name="graphics_Temperature"/>
      </item>
      <item row="1" column="0">
       <widget class="QChartView" name="graphics_Static"/>
      </item>
      <item row="1" column="1">
       <widget class="QChartView" name="graphics_Shear"/>
      </item>
      <item row="1" column="2">
       <widget class="QChartView" name="graphics_Strain"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_49" stretch="10,1,1">
      <item>
       <spacer name="horizontalSpacer_44">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="add_Row">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_Row">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>参数设置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,5,1,5,1,5,1">
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>91</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>设备IP：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_2"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>91</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>端口号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_9"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>91</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>设备名称：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_10"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>91</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,3,1,3,1,3,1,3,1">
         <item>
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>27</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>连接设备</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>自动获取</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>27</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_StartAndStop">
           <property name="text">
            <string>开始采集</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_17">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>27</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Clear">
           <property name="text">
            <string>数据清理</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_18">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>27</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="home"/>
   <addaction name="clean"/>
   <addaction name="connect"/>
   <addaction name="setting"/>
   <addaction name="monitor"/>
   <addaction name="data"/>
   <addaction name="more"/>
  </widget>
  <action name="clean">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/33.png</normaloff>:/logo/33.png</iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
   <property name="toolTip">
    <string>清除</string>
   </property>
  </action>
  <action name="home">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/24.png</normaloff>:/logo/24.png</iconset>
   </property>
   <property name="text">
    <string>目录</string>
   </property>
   <property name="toolTip">
    <string>目录</string>
   </property>
  </action>
  <action name="connect">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/8.png</normaloff>:/logo/8.png</iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="toolTip">
    <string>连接</string>
   </property>
  </action>
  <action name="setting">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/3.png</normaloff>:/logo/3.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
  </action>
  <action name="monitor">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/13.png</normaloff>:/logo/13.png</iconset>
   </property>
   <property name="text">
    <string>监测</string>
   </property>
   <property name="toolTip">
    <string>监测</string>
   </property>
  </action>
  <action name="data">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/15.png</normaloff>:/logo/15.png</iconset>
   </property>
   <property name="text">
    <string>数据</string>
   </property>
   <property name="toolTip">
    <string>数据</string>
   </property>
  </action>
  <action name="more">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/28.png</normaloff>:/logo/28.png</iconset>
   </property>
   <property name="text">
    <string>更多</string>
   </property>
   <property name="toolTip">
    <string>更多</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/main.qrc"/>
 </resources>
 <connections/>
</ui>
